python/ray/util/state/state_manager.py

```diff
@@ -379,9 +379,19 @@ async def get_job_info(self, timeout: int = None) -> List[JobDetails]:
 
         return list(driver_jobs.values()) + submission_jobs
 
-    async def get_all_cluster_events(self) -> Dictionary:
+    async def get_all_cluster_events(
```

Comment:
Just remove this?

Reply:
Yes, we could remove this if we would like to remove v0 api